shop-be/product-service/serverless.ts

```diff
@@ -18,11 +21,16 @@ const serverlessConfiguration: AWS = {
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
       NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+      POSTGRESQL_HOST: process.env.POSTGRESQL_HOST,
```

Comment:
Usually all process.env variables move to the dedicated config file at the root of the src. And all handlers do not work with process.env directly, but with config object.

// config.js

export default {
  POSTGRESQL_HOST: process.env.POSTGRESQL_HOST,
.....
};

Reply:
added config.ts and connection string